src/Symfony/Component/Mime/MimeTypes.php

```diff
@@ -1090,7 +1099,7 @@ public function guessMimeType(string $path): ?string
         'application/x-sh' => ['sh'],
         'application/x-shar' => ['shar'],
         'application/x-shared-library-la' => ['la'],
-        'application/x-sharedlib' => ['so'],
+        'application/x-sharedlib' => ['so', 'so.[0-9]*'],
```

Comment:
I don't think the class supports patterns in the way it uses this array.

Reply:
Done

Reply:
This should be solved by excluding extensions containing a range pattern (can probably be detected based on the presence of `[`) in the update script, not by manual cleanup. Otherwise, we will have the same issue in the next update (and maybe I won't be the one reviewing it to notice it)

Reply:
> This should be solved by excluding extensions containing a range pattern (can probably be detected based on the presence of `[`) in the update script, not by manual cleanup. Otherwise, we will have the same issue in the next update (and maybe I won't be the one reviewing it to notice it)

As we update this file every two years, let's keep it simple.

Reply:
The automatic check is probably just adding a check for the presence of `[` in the pattern in https://github.com/symfony/symfony/blob/139419d2b540fdee671b4012dc4fe318f314721f/src/Symfony/Component/Mime/Resources/bin/update_mime_types.php#L31

Reply:
But what about `anim[1-9j]` also in this file that we can handle?

Reply:
What about just doing it manually when needed then?

Reply:
"when needed" is "each time the script is run", given that the script will override previous manual changes.